astronomy/sol_gravity_model.proto.txt

```diff
@@ -276,8 +280,1429 @@ gravity_model {
     axis_declination        : "66.5392 deg"
     reference_angle         : "38.3213 deg"
     angular_frequency       : "13.17635815 deg / d"
-    j2                      : 2.0321568464952570e-04
     reference_radius        : "1738.0 km"
+    geopotential {
+      row {
+        degree: 2
+        column {order:  0, cos: -9.0884339347424299e-05, sin:  0.0000000000000000e+00},
```

Comment:
Could we use `0` for the zeroes, like we did for the Earth?

Reply:
Done.